doc/sphinx/developer_manual/start_devel_and_contrib/tools/astyle.md

```diff
@@ -2,6 +2,16 @@
 Astyle
 =====
 
-```{todo}
-Explain what Astyle does, why it is important and how to get the right version.
-```
\ No newline at end of file
+Astyle (Artistic Style) is a tool that automates the formatting of code. Executing astyle will automatically indent new code to the standard that the Geodynamic WorldBuilder uses, and should be done before each pull request to ensure that the indenter tests do not fail. 
+
+GWB requires the use of astyle v2.04, which can be downloaded from sourceforge [here](https://sourceforge.net/projects/astyle/files/astyle/astyle%202.04/) for mac, linux or windows. 
+
+An example one line command for linux builds is shown below, followed by a more general description on building astyle and linking it to the GWB.
```

Comment:
What do you think of:  `An example one line command for to build and install the correct astyle on linux is shown below, followed`